### PR TITLE
Readability of Figures: Fig 11 and Fig 16 adjustments.

### PR DESCRIPTION
**Pull request by williamdemeo**
# Description

This closes issue #501.

# Checklist

- [X] Commit sequence broadly makes sense and commits have useful messages
- [X] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [X] Self-reviewed the diff

### CODE REVIEW

File: src/Ledger/Gov.lagda [hunk 1]
```diff
@@ -110,13 +110,13 @@ validHFAction _ _ _ = ⊤
 \footnotetext{\AgdaBound{l}~\AgdaFunction{∷ʳ}~\AgdaBound{x} appends element \AgdaBound{x} to list \AgdaBound{l}.}
 \begin{figure*}[h]
 \begin{AgdaMultiCode}
-\begin{code}[hide]
--- convert list of (GovActionID,GovActionState)-pairs to list GovActionID pairs.
+\begin{code}
```
Comment (WhatisRT) [inline src/Ledger/Gov.lagda:113]: This isn't what we wanted, this stuff was intentionally hidden (you've even proposed that). So the figure should be fixed without showing that again.

Comment (williamdemeo) [reply src/Ledger/Gov.lagda:113]: Yes, I even proposed that, but in response to that proposal you said we would have to add an explanation to the prose if we hide this code, which I thought was a worse option than simply showing the code with a comment.

Comment (williamdemeo) [reply src/Ledger/Gov.lagda:113]: I've gone ahead and removed the functions and added a paragraph describing the figure and mentioning the hidden function and their purpose.

**Review by WhatisRT — APPROVED** [review]
LGTM

**Pull request merged**